README.md

```diff
@@ -277,6 +277,18 @@ execute the generated binaries using QEMU:
     $RISCV/bin/clang++ -march=rv64imafdc -stdlib=libc++ -o hello_world_cpp hello_world_cpp.cxx
     $RISCV/bin/qemu-riscv64 -L $RISCV/sysroot ./hello_world_cpp
 
+Build newlib toolchain and run examples:
```

Comment:
Add some comment to mention clang/LLVM isn't work with `--with-multilib-generator=`

Reply:
Thanks, it's done.